x-pack/plugins/maps/public/embeddable/map_embeddable_factory.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+import chrome from 'ui/chrome';
+import { EmbeddableFactory } from 'ui/embeddable';
+import { MapEmbeddable } from './map_embeddable';
+import { indexPatternService } from '../kibana_services';
+
+export class MapEmbeddableFactory extends EmbeddableFactory {
+
+  constructor(gisMapSavedObjectLoader) {
+    super({ name: 'map' });
+    this.savedObjectLoader = gisMapSavedObjectLoader;
+  }
+
+  getEditPath(mapId) {
```

Comment:
shouldn't be a method on the prototype. Make either static, or extract in simple function. or, since it's only used once, remove alltogether